sdcm/sct_config.py

```diff
@@ -768,7 +768,10 @@ class SCTConfiguration(dict):
              """,
              choices=("keep", "keep-on-failure", "destroy")),
         dict(name="workaround_kernel_bug_for_iotune", env="SCT_WORKAROUND_KERNEL_BUG_FOR_IOTUNE", type=bool,
-             help="Workaround a known kernel bug which causes iotune to fail in scylla_io_setup, only effect GCE backend")
+             help="Workaround a known kernel bug which causes iotune to fail in scylla_io_setup, only effect GCE backend"),
+
+        dict(name="loader_swap_size", env="SCT_LOADER_SWAP_SIZE", type=int,
+             help="Swap file size in bytes calculated by x * 1MB")
```

Comment:
put a comma after your line so when next param will be added one will not need change unnecessary code

Reply:
ok.